modules/javafx.graphics/src/main/docs/javafx/scene/doc-files/cssref.html

```diff
@@ -680,6 +682,80 @@ <h3><a id="introinheritance">Inheritance</a></h3>
         inherited provided there is not an author or an inline-style that applies specifically to that node.
         In this case, the inherited font is created from the user-set font and any parts of the applicable
         author or in-line style.
+
+    <h3><a id="introtransitions">Transitions</a></h3>
+    <p>JavaFX supports <em>implicit transitions</em> for properties that are styled by CSS. When a property value is
+        changed, it smoothly transitions to the new value over a period of time. Implicit transitions are supported
+        for all primitive types, as well as for types that implement <code>javafx.animation.Interpolatable</code>.</p>
+    <p>Transitions can be defined for any node in the JavaFX scene graph with the following properties:</p>
```

Comment:
The way this is phrased makes it sound like the node has "the following properties", not the transition. Maybe move that part:
"Transitions with the following properties can be defined for any node in the JavaFX scene graph", or just add a comma.

Reply:
I understand that you're saying that `property`, `duration`, `timing-function`, and `delay` are all sub-properties of `transition`.

However, from a CSS perspective, `transition-property`, `transition-duration`, `transition-timing-function` and `transition-delay` are properties of `Node`, in the same way as `-fx-background-color`, `-fx-background-insets`, etc. are properties of `Node`.

`transition` is a short-hand property that combines all four properties into one (we don't have a short-hand property for backgrounds yet). I think that both mental models are basically correct (four properties of node, vs. four sub-properties of transition).

Reply:
I understand. I find it a bit confusing, but OK to leave as is.